tools/runners/run-all.js

```diff
@@ -163,6 +178,8 @@ class Runner {
 
     if (! self.stopped) {
       buildmessage.enterJob({ title: "starting your app" }, function () {
+        self.errorAppRunner.start();
```

Comment:
I didn't know what the otherRunners array does - if it can fit there, then that works for me!